flag_cleanup.py

```diff
@@ -153,8 +160,15 @@ def cleanup_replace_node(replace_node):
     buffer_size_bytes = 10 * 1024 * 1024  # 10 megabytes
     repo.config_writer().set_value("http", "postBuffer", buffer_size_bytes).release()
 
-    g = Github(github_token)
-    user = g.get_user(github_username)
+    if git_username == "":
+        g = Github(github_token)
+        user = g.get_user(github_username)  # Why is this never used?
```

Comment:
can't remember if that was to check the token worked. or if we used user for something previously, but no longer need it.